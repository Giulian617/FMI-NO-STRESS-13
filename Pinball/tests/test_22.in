100 30886120975325
33 45 7
68 45 6
24 68 10
69 24 10
37 33 3
32 45 5
81 69 6
34 45 9
47 69 5
75 69 4
10 24 2
55 81 6
97 55 10
9 34 9
54 81 4
99 97 2
100 32 4
50 47 8
22 100 3
49 54 7
6 45 2
73 10 8
60 97 6
59 75 7
19 32 3
13 73 10
88 13 2
58 34 6
71 13 5
48 19 2
27 9 6
42 99 4
77 48 6
40 49 3
82 27 6
94 69 3
4 60 8
90 73 10
15 6 9
83 71 2
52 15 9
53 52 6
93 27 8
31 45 9
23 77 4
79 33 4
26 23 7
16 82 8
76 9 5
29 31 5
51 10 9
5 23 8
8 58 6
28 100 9
11 59 5
87 59 9
85 75 6
41 71 7
91 51 8
39 50 7
92 23 5
89 53 6
65 48 6
66 73 4
3 51 3
12 29 6
56 68 10
95 53 6
18 91 10
62 6 4
44 24 7
7 95 3
30 12 5
2 95 5
74 50 9
70 27 8
17 75 10
78 94 6
20 23 7
80 85 5
38 77 5
25 8 8
63 50 8
98 42 10
14 6 7
36 20 6
43 97 7
21 43 10
96 7 10
64 74 4
46 32 10
57 8 2
1 4 2
86 29 9
61 6 8
84 1 7
72 16 9
67 62 2
35 57 2
