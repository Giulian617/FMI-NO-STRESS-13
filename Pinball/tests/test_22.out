573298726
750005001
854390631
587798126
715075525
357891085
517147913
256856325
435985613
864792964
830953682
504801403
640705274
509803694
830953682
264458113
381802054
773729759
796943133
888950802
67665641
212008668
602576701
41802550
57228346
900738493
431180887
134323236
368229043
507226451
345162116
465869861
260574895
557869774
1097368
902664710
958540675
934710514
690979882
97785997
717976671
766515446
516871410
507226451
159223522
457607356
